python/hsfs/feature_view.py

```diff
@@ -657,10 +657,10 @@ def get_inference_helpers(
                 If set to False, the online feature store storage connector is used
                 which relies on the private IP. Defaults to True if connection to Hopsworks is established from
                 external environment (e.g AWS Sagemaker or Google Colab), otherwise to False.
-            return_type: `"pandas"` or `"dict"`. Defaults to `"dict"`.
+            return_type: `"pandas"`, `"polars"` or `"dict"`. Defaults to `"dict"`.
 
         # Returns
-            `pd.DataFrame` or `List[dict]`.  Defaults to `pd.DataFrame`.
+            `pd.DataFrame`, `polars.DataFrame` or `List[dict]`.  Defaults to `pd.DataFrame`.
```

Comment:
It says ` Defaults to `pd.DataFrame`. but function says `dataframe_type: Optional[str] = "default"`. as above set it correctly and explain what default is

Reply:
in `get_inference_helpers` function the default value set for `return_type` earlier was "pandas". That is why it is mentioned as pd.Dataframe. I corrected the `return_type` comment as  Defaults to `"pandas"`.